services/worker/src/worker/job_runners/split/duckdb_index.py

```diff
@@ -205,9 +307,12 @@ def compute_split_duckdb_index_response(
             create_index_sql = CREATE_INDEX_COMMAND.format(columns=indexable_columns)
             logging.info(create_index_sql)
             con.sql(create_index_sql)
+
     finally:
         con.close()
 
+    features["__hf_index_id"] = {"dtype": "int64", "_type": "Value"}
```

Comment:
I don't update features with newly created transformed columns to avoid including them in search/filter response

Reply:
It looks to be duplicated here https://github.com/huggingface/dataset-viewer/pull/2737/files#diff-d9a2c828d7feca3b7f9e332e040ef861e842a16d18276b356461d2aa34396a8aR398